delta-lake/src/main/java/org/apache/iceberg/delta/BaseSnapshotDeltaLakeTableAction.java

```diff
@@ -412,6 +426,25 @@ private Map<String, String> destTableProperties(
     return additionalPropertiesBuilder.build();
   }
 
+  private void tagCurrentSnapshot(long deltaVersion, Transaction transaction) {
+    long currentSnapshotId = transaction.table().currentSnapshot().snapshotId();
+
+    ManageSnapshots manageSnapshots = transaction.manageSnapshots();
+    manageSnapshots.createTag(DELTA_VERSION_TAG_PREFIX + deltaVersion, currentSnapshotId);
+
+    Timestamp deltaVersionTimestamp = deltaLog.getCommitInfoAt(deltaVersion).getTimestamp();
```

Comment:
should we also check `deltaLog.getCommitInfoAt(deltaVersion)` is not null?

Reply:
Based on its source code, it seems this method returns non-null result:
https://github.com/delta-io/connectors/blob/f11c355649bf1182dd74b563bc399fa7f1b7fe97/standalone/src/main/scala/io/delta/standalone/internal/DeltaLogImpl.scala#L101-L104

https://github.com/delta-io/connectors/blob/f11c355649bf1182dd74b563bc399fa7f1b7fe97/standalone/src/main/scala/io/delta/standalone/internal/util/ConversionUtils.scala#L126-L159

So I think we are safe to exclude null-check for simplification. However, given that the interface of this method does not specify it as `nonnull`, I am not against adding a null check. What do you think?

Reply:
cool, in that case I am fine as is!